cmake/FDBComponents.cmake

```diff
@@ -121,7 +121,25 @@ endif()
 
 # TOML can download and install itself into the binary directory, so it should
 # always be available.
-find_package(TOML11)
+#find_package(TOML11)
```

Comment:
The toml PR was merged with `#find_package(TOML11)` removed, but I _think_ this might end up sticking around if we merge this as is. I think we should probably remove the toml changes from this PR

Reply:
I removed it